src/main/java/org/jabref/logic/crawler/StudyFetcher.java

```diff
@@ -48,7 +48,7 @@ private QueryResult getQueryResult(String searchQuery) {
     }
 
     /**
-     * Queries all Databases on the given searchQuery.
+     * Queries all Catalogs on the given searchQuery.
```

Comment:
Please also fix casing when you are on it.

```suggestion
     * Queries all catalogs on the given searchQuery.
```